src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/ProjectSystem/ProjectConfigurationFilePathChangedEventArgs.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.CodeAnalysis.Razor.ProjectSystem
+{
+    internal class ProjectConfigurationFilePathChangedEventArgs : EventArgs
+    {
+        public ProjectConfigurationFilePathChangedEventArgs(string projectFilePath, string configurationFilePath)
+        {
+            if (projectFilePath is null)
+            {
+                throw new ArgumentNullException(nameof(projectFilePath));
+            }
```

Comment:
null check needed for `configurationFilePath`?

Reply:
Nope, you can have a `null` configuration path when the project configuration publish path is "removed"